app/views/reducers/_form.html.erb

```diff
@@ -35,7 +35,8 @@
         <%= filters.input :from %>
         <%= filters.input :to %>
         <%= filters.input :extractor_keys, placeholder: '["list", "of", "keys"]' %>
-        <%= filters.input :repeated_classifications, collection: ::Filters::FilterByRepeatedness::REPEATED_CLASSIFICATIONS, selected: 1 %>
+        <%= filters.input :repeated_classifications, collection: ::Filters::FilterByRepeatedness::REPEATED_CLASSIFICATIONS %>
```

Comment:
Any reason why the `selected: 1` was removed?

Reply:
Yes -- where "defaults" meant "selected option"

> This PR also changes the filter form behavior with regards to defaults: previously, the keep_first option for filtering on repeated classifications was selected by default. This default choice was not enabled on reducer creation, only after its first update via the already-selected form entry. Here, I adopt the creation default for both repeatedness and training behavior filters: no filtering.